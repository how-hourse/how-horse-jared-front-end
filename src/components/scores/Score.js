import React from 'react';
import PropTypes from 'prop-types';

function Score({ score }) {
  const divStyle = {
    display: 'flex',
    justifyContent: 'space-between',
    padding: '3px'
  };

  return (
    <div style={divStyle}>
      <p>{score.user}</p>
      <p>{score.score}</p>
    </div>
  );
}

Score.propTypes = {
  score: PropTypes.object.isRequired
};

export default Score;
