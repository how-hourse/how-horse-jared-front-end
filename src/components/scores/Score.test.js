import React from 'react';
import { shallow } from 'enzyme';
import Score from './Score';

describe('Score component', () => {
  it('renders a Score', () => {
    const score = {
      name: 'terran',
      score: 200
    };
    const wrapper = shallow(<Score score={score} />);
    expect(wrapper).toMatchSnapshot();
  });
});
