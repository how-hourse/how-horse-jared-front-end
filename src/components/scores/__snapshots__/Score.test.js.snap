// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Score component renders a Score 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "justifyContent": "space-between",
    }
  }
>
  <p>
    terran
  </p>
  <p>
    200
  </p>
</div>
`;
